src/RuleMatchedLog.php

```diff
@@ -261,31 +227,26 @@ public function showItemForm($items_id, $itemtype)
             'START'  => (int)$start,
             'LIMIT'  => (int)$_SESSION['glpilist_limit']
         ];
+        $rows = [];
         foreach ($DB->request($params) as $data) {
-            echo "<tr class='tab_bg_1'>";
-            echo "<td>";
-            echo Html::convDateTime($data['date']);
-            echo "</td>";
-            echo "<td>";
-            if ($rule->getFromDB($data['rules_id'])) {
-                echo $rule->getLink(1);
+            $row = [
+                'date' => $data['date'],
+                'rules_id' => $data['rules_id'],
+                'agents_id' => $data['agents_id'],
+                'modulename'=> Request::getModuleName($data['method']),
+            ];
```

Comment:
```suggestion
            $row = [
                'date'          => $data['date'],
                'rules_id'      => $data['rules_id'],
                'itemtype'      => $data['itemtype'],
                'items_id'      => $data['items_id'],
                'modulename'    => Request::getModuleName($data['method'])
            ];
```

Reply:
One particular reason for not displaying the criteria here (on the agents page)?

Reply:
this would make it possible to have a single template

Reply:
Cedric suggested above to make 2 templates